src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/TemplateCollectionFunctionalTests.cs

```diff
@@ -228,9 +222,11 @@ public async Task GiveImageReference_WhenGetTemplateCollection_ACorrectTemplateC
         }
 
         [Theory]
-        [MemberData(nameof(GetHl7v2DataAndTemplateImageReference))]
+        [MemberData(nameof(GetHl7v2DataAndEntryTemplate))]
         public async Task GetTemplateCollectionFromACR_WhenGivenHl7v2DataForConverting__ExpectedFhirResourceShouldBeReturnedAsync(string hl7v2Data, string entryTemplate)
         {
+            hl7v2Data = Path.Combine(_sampleDataDirectory, "Hl7v2", hl7v2Data);
```

Comment:
Combine "Hl7v2" in `GetHl7v2DataAndEntryTemplate`. Example could be found in `FunctionalTests.cs`.